query/README.md

```diff
@@ -0,0 +1,146 @@
+Keen Query Clients
+===================
+
+[![Build Status](https://travis-ci.org/keenlabs/KeenClient-Java.png?branch=master)](https://travis-ci.org/keenlabs/KeenClient-Java)
+
+The query capabilities within the Java Keen client enable you to send POST queries and receive the results of the queries in a JSON object. For query types, refer to API technical reference: https://keen.io/docs/api/reference/.
+
+## Usage
+### Building a Keen Query Client
+You can build a KeenQueryClient by just providing a KeenProject.
+```java
+KeenProject queryProject = new KeenProject("<project id>", "<write key>", "<read key>");
```

Comment:
Nit: A casual reader of these docs might wonder why the write key is being provided when this is a query client. It might be worth a quick comment explaining that only the read key is used by the query client, and that it's OK and normal to provide a `null` write key when constructing the project (unless the same project object is also going to be used for publishing events).